services/spar/src/Spar/Scim/User.hs

```diff
@@ -542,7 +516,7 @@ deleteScimUser ScimTokenInfo {stiTeam} uid = do
           $ BrigTypes.userSSOId brigUser
       uref <- either logThenServerError pure $ Brig.fromUserSSOId ssoId
       lift . wrapMonadClient $ Data.deleteSAMLUser uref
-      lift . wrapMonadClient $ Data.deleteScimUser uid
+      lift . wrapMonadClient $ Data.deleteScimUserTimes uid
```

Comment:
what happens if this crashes here?

Reply:
can you add a comment [here](https://github.com/zinfra/backend-issues/issues/1631) and we'll fix it later?